firebase_admin/dynamic_links.py

```diff
@@ -94,54 +111,45 @@ def event_stats(self):
         """Returns the event statistics for this link, for the requested period.
 
         Returns:
-          event_stats: A list of `EventStats`.
+          event_stats: A list of ``EventStats``.
         """
         return self._stats
 
 class EventStats(object):
-    """`EventStats` is a single stat item containing (platform, event, count)"""
-
-    _platforms = {
-        'DESKTOP': PLATFORM_DESKTOP,
-        'IOS': PLATFORM_IOS,
-        'ANDROID': PLATFORM_ANDROID
-    }
-
-    _event_types = {
-        'CLICK': EVENT_TYPE_CLICK,
-        'REDIRECT': EVENT_TYPE_REDIRECT,
-        'APP_INSTALL': EVENT_TYPE_APP_INSTALL,
-        'APP_FIRST_OPEN': EVENT_TYPE_APP_FIRST_OPEN,
-        'APP_RE_OPEN': EVENT_TYPE_APP_RE_OPEN
-    }
-
-    def __init__(self, platform, event, count):
-        """Create new instance of EventStats(platform, event, count)"""
-        if isinstance(platform, six.string_types) and platform in self._platforms.keys():
-            raise ValueError(('Raw string "{}" detected. Use a dynamic_links.PLATFORM_* constant' +
-                              ' or the make_from_strings() method.').format(platform))
-        if not isinstance(platform, six.string_types) or platform not in self._platforms.values():
-            raise ValueError('platform {}, not recognized'.format(platform))
-        self._platform = platform
-
-        if isinstance(event, six.string_types) and event in self._event_types.keys():
-            raise ValueError(('Raw string {} detected. Use one of the dynamic_links.EVENT_TYPES_' +
-                              ' constants, or the make_from_strings() method.').format(event))
-        if not isinstance(event, six.string_types) or event not in self._event_types.values():
-            raise ValueError('event_type {}, not recognized'.format(event))
-        self._event = event
-
-        if not isinstance(count, int) or isinstance(count, bool) or count < 0:
-            raise ValueError('Count: {} must be a non negative int'.format(count))
-        self._count = count
-
-    @classmethod
-    def make_from_strings(cls, platform, event, count):
-        """make_from_strings creates an EventStat object given the appropriate constants. e.g:
-        make_from_strings(platform=PLATFORM_DESKTOP, event=EVENT_TYPE_REDIRECT, count=4)"""
-        return EventStats(cls._platforms[platform],
-                          cls._event_types[event],
-                          int(count))
+    """``EventStat`` is a single stat item containing (platform, event, count)"""
+
+    def __init__(self, **kwargs):
```

Comment:
Please document the class rather than the constructor. That's what developers will care about. Constructor is really an internal API.